src/plugins/auto/README.md

```diff
@@ -0,0 +1,21 @@
+# OpenVINO AUTO Plugin
+
+AUTO plugin for OpenVINO™ Runtime, uses a “virtual” or a “proxy” device, which does not bind to a specific type of hardware, but rather selects the processing unit for inference automatically. It detects available devices, picks the best-suited for the task, and configures optimization settings. This way, developer can write the application once and deploy it anywhere. 
+
+## Components
+Please see more detail features at [AUTO Plugin introduction](https://docs.openvino.ai/latest/openvino_docs_OV_UG_supported_plugins_AUTO.html).
+
+## Support
+Please report issues and suggestions [GitHub issues](https://github.com/openvinotoolkit/openvino/issues).
+
+## How to Contribute
+We welcome community contributions to AUTO plugin. If you have an idea how to improve the library:
+
+Share your proposal via [GitHub issues](https://github.com/openvinotoolkit/openvino/issues)
```

Comment:
I think here can be a link to Contribution guide

Reply:
Updated